src/cmd.ts

```diff
@@ -13,27 +14,43 @@ var program = require('commander');
 // set up sdsAccess
 
 
-async function uploadAndRunAll(loginData: config.LoginData, folder: string, prefix: string): Promise<sdsAccess.scriptT[]> {
+async function uploadAndRunAll(loginData: config.LoginData, files: sdsAccess.scriptT[]): Promise<sdsAccess.scriptT[]> {
```

Comment:
I'm pretty sure that the whole script actually was implemented for me to execute and upload scripts from the command line. Also, this function is not exported, so the only place where this function can be used is in the same script. But you're right, it would be better to split this into multiple functions.